submissions/dragalur/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" >
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" >
+		<title>Document</title>
+		<link rel="stylesheet" href="style.css" >
+	</head>
+	<body>
+		<div class="wrapper">
+			<div class="header">
+				<nav class="menu">
+					<ul class="menu-list">
+						<li>
+							<a class="menu-link"  href="">Mail</a>
+						</li>
+						<li>
+							<a class="menu-link" href="">Image</a>
+						</li>
+						<li class="focuses">
+							<label for="navbar-button" class="navbar-button">
+								<img class="image-poup" src="Image/popup-button.png" alt="" >
+							</label>
+						</li>
+						<li>
+							<a class="menu-link" href="">
+								<img src="Image/bell.png" width="24" alt="" >
+							</a>
+						</li>
+						<li>
+							<a class="menu-link" href="https://github.com/dragalur">
+								<div class="menu-item-personLink"></div>
+							</a>
+						</li>
+					</ul>
+				</nav>
+			</div>
+
+			<input 
+				tabindex="1"
```

Comment:
All interactive elements have a focus by default, use `tabindex` attribute for adding a noninteractive element to the focusable elements or for changing focus order. [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex).